Modules/Filtering/ImageFeature/test/itkHessianRecursiveGaussianFilterScaleSpaceTest.cxx

```diff
@@ -165,9 +163,7 @@ itkHessianRecursiveGaussianFilterScaleSpaceTest(int, char *[])
     PointType center;
     center.Fill(0.0);
 
-    IndexType centerIndex;
-
-    outputImage->TransformPhysicalPointToIndex(center, centerIndex);
+    IndexType centerIndex = outputImage->TransformPhysicalPointToIndex(center);
```

Comment:
I am not sure how important performance is here, and other similar instances. But this improves style slightly, and saves a line of code.